server/controllers/tController.js

```diff
@@ -0,0 +1,99 @@
+const { Todo } = require("../models");
+
+class tController {
```

Comment:
namingnya kok tController 
```suggestion
class todoController {
```

Reply:
tController maksudnya todoController kak
baiknya todoController aja kah?
sama kayak uController itu userController kak

Reply:
ntar ku refactor deh, thank you kak